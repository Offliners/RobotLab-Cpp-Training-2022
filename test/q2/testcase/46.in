817
CART
boWl
hAirdresser
cancer
PROPOSE
SUpERimPose
JOKE
DRAMATIc
PRESENCE
irritAte
persIsT
LOTTERY
PoWERFUL
cross
title
proMote
nUdE
jaR
7I5h
paste
Per
CONVERTIBLE
tirEd
mail
METAPHOR
manNers
1ELIVER
InFInITE
Assign
dRawiNg
SURGeoN
BloOm
totalLy
FrIendship
slender
CONSERVaTION
AbOve
tRANSMiSSION
HYPHEN
1922t
COCKTAIL
8L9079S
hierarchy
inhaBItanT
sOMEDAY
646iT71n
quaNtitY
aUTOMATIC
R
TouGh
BUREAUCRACY
ReBEl
humiliaTioN
TASTE
SUMMON
POSSIBiLITY
INvEntor
cOuch
BoOSt
PrOCEeD
cLOseT
aHEad
sUckle
correct
BRIGADE
rElieVe
STraIgHteN
tUmpike
sUFFerIng
PAnda
assumE
gEogRAPHIcAl
REsOlUTIOn
preliminary
COmmaNd
seAt
eXAMineE
beggar
sTrUGGlE
fLOAT
NURSe
disease
BEHOLD
79MP0
E5vIo7S
PREfErablY
7aP
gulf
w542183
LOSer
s
CANaL
borinG
GRAIN
Away
RESORT
plUs
SOUL
courtyard
vaCation
RESIST
CorRelAtion
edit
entail
SoLo
COmpLy
Di45153
TRaFfIC
Unofficial
REviEw
LivINg
sKiRt
3H82E
OuTCoMe
eXCEPT
goRgeoUS
lAWmAkER
MODER4T6
AWKwaRDlY
hAlf
StaPLE
DIE
whoM
TAPE
salaD
duck
vIRus
mOAN
DICTATIon
pHILOSopHY
clOThes
cope
emBarrAsS
ImMENSe
PhotogRaPhY
cUCUMBER
Ev2nINg
GHOST
margiNal
Humid
aspeCt
pajamAs
ANgEl
sTiMULATION
liKELy
BaThROoM
RESuME
pAINFUL
land
LITERACY
NOTICEABLE
drunkard
trAy
aGoNy
FountAIn
nOTiCE
HEADAChE
c3N5I484L
dEficiT
piratE
PEER
PePpER
FUriOUs
REUNION
pLanet
breathTaKing
CARP
fReshMan
dash
f7e4
dIsh
diPLomat
HaBIt
6RO2O4T
farmer
bore
EQUIvALENT
TrAnSPlANt
LiquOR
P332
POlitE
cult
inSTrUmEnt
MRs.
PLEA
spRinG
OUtdOoRs
ASCRIBE
isOlatIoN
SUDdeN
DeMAnd
chErRy
reServation
aIrWay
EVOlVE
GOSpEL
FactorY
WhoSe
aPpRaiSAL
rECoRD
approach
saLvatIOn
Drunk
camel
CRITIc
VivID
SyNONyM
DARK
BoOK
AlTERN579V0
GROWTH
hURl
STALL
aCt
PrOfESSOR
DEtAILED
REFUGEE
Blur
rEcOGnITion
hOPEful
PERSONALITY
TraditiONal
laMa
LeNgTHY
reGard
COp
exPensIVe
COnneCT
sperm
homely
BriEfcaSe
moBile
TraGIc
TIMID
sun
beautifuL
ReFraIn
1EdiCAT85N
lYRIC
saturDaY
HA5SH
grIEve
TOwEL
MORTAL
praiSeworThy
PEe1
CoUNTerPart
ConSOle
INn
dEsCeNDanT
pictUresqUe
cULMInAte
huNt
shooT
ALL
amazINg
bewilder
never
toucH
soUTH
blEaK
sedUce
angry
rE7r6T
UNeMPloyEd
now
mUsical
AGINg
COMPaCt
Reverse
excHANge
TRANSfER
cr3CK
COUNcIl
aDDICTION
TENSION
3ER1Y
0CEBeR0
consequent
FAIl
7eT
MILeAGE
PRESIDE
CALm
BROCHure
EXPERIenCE
30225E
TARIFF
trAnSlate
alarm
ALonGsidE
beach
ruDe
sCRATCH
MARgin
FUNDAMENTAL
48938249C0
cOnsidERaTIOn
FROST
PIcTURE
MOnUMENt
HArboR
0e4LE3t
DIvERSITY
TIGHT
neaRly
Runner
thYroid
HObBY
GeAR
compete
counsellor
EMOTION
sMell
EFfICIENT
aTtRacT
BALLROOM
fuLfill
complexion
presEntation
MATTER
SPOUse
27Ag83Fl9
0N31N12070
union
join
bLOsSOM
eXTeRIOR
UPBRINGING
ABOLITIoN
exHIbIT
sCoUt
SaLVAGE
lightNing
HUMILiATion
HETeROseXUaL
annouNce
AlIE0
beyonD
562i6368
usE
O
VARIABLE
2O9
musTard
REPENT
hAUL
just
HuMID
GENTLE
r4c1aL
SensibLe
Trophy
cOAsTgUArD
INEFFECTIVE
DIrT7
BAKE
ConTInEnt
MeRCy
SwinG
lIMb
essay
29m0ng
j8IN
basket
depREciate
meditaTiOn
SELf
Isolate
o8IDA4Ion
crEATURE
5ra9sla7ion
climB
villain
drown
mirACLe
EsTeem
luxurY
SNEEZe
ULTImAtELY
CROWd
cUshion
peTition
MOTHER
coarse
bouleVArd
sIgnIficaNT
boOTh
JuDgE
Statue
ConFIdENTIAL
LUnCH
VacCIne
jUNCTiON
scandal
grill
dIsasTrous
reach
STRaNGe
OBSERVAtIOn
pint
FAVor
cRedibILitY
DISmAY
BuFFet
SUBSEQUENTLY
FurTHERMOre
diVerSiFY
ERAsER
weapon
9u894a764allY
INDECISION
pEa
56859a40
GUM
atTeNtioN
RESPECTABLE
pRoUDLy
tHEoRY
PLENTIFuL
track
temPorary
THIRD
oVERheAR
MiSBEhAvE
aircraft
POunD
TEST
NONETH7LEsS
cHIP
oUTbReAK
aVeNUE
democrat
raInBow
hO4sewor7
BAthe
batH
diFFUSiOn
coOkER
accordiNgly
easT
skip
week
IrOny
COLLISIOn
RAINY
diplomacy
SUNseT
GRACEFUl
invest
DInOSaur
spare
leveL
MaJesTy
dIalEcT
logic
OWNER
lANDlorD
diffErenCe
RESErvOiR
tRaNsiTiOn
Bass
JUMP
SYMBOL
1a53
desTructive
EARNEST
sYLlABLE
PoLish
AuTUmN
FoundATiOn
hostes7
vULgar
Universe
deLIcious
horrible
dAmaGE
shaft
6END
FOnD
direction
8O09
DEPart
about
rAilRoad
RAPID
BOND
HuNTEr
contradictiOn
sHEeP
fEbRuAry
TOKEN
TeLEvISION
06T165r
gLORiOUs
pErmission
DisgusTIng
OPPOnENT
SYnTheSIS
shelf
ACcomMODaTion
CRuEL
CiRCuS
iRrESponsible
veRd844
EFfECt
lifeboat
ridiculous
5ounde5
cHOiCe
3pT3C57
OvERwORk
pErpetUal
tHuS
12p462a9
i.q.
BUFFET
LOUNGE
daRe
OVERSEas
dIFFeR
eYESight
ATHLETIC
MiLiTaRy
caress
DOdgE
antique
MArvEl
DeBT
3ov1lY
domiNant
T2P6
INeXpenSIVE
ReLaTE
wANDER
cHaIr
PerSiSTENt
TERRIFIED
8C59L0T
PrEsUMPTIOn
coNVENTiONAL
interesTing
COMER
wHAt
C
baG
BAsTarD
lucky
l
CONcESSION
REGAIN
MELLOw
52r019
MAteriAL
cane
coopERAtIve
chinA
ideoLogicaL
28s7L71974
iNNOceNt
ENEMY
5iP1o12
PHoTOgrApH/PhOTO
NOVEmbER
coNVIct
9a06n21
magIcian
cAfE
2N1I5ING
PEACH
EncYclOPEDIA
site
defeNsiVe
ReStOrAtIoN
aMBaSSadOR
immune
demonstration
7Dvance
sUPErViSe
NEVeRThelESS
napkin
AsseRtiOn
apPearancE
ChildiSh
lENs
CATASTROPHE
ProsEcutIoN
2heN26en8n
Nice
sTANDARDIZE
aCceSsoRy
efficIeNcy
ELABORATE
tuNNel
ShaPe
street
sponsor
7a5
noBLE
manhooD
intersecTioN
PREVIOUS
SEpARATELY
7o377
tre5p1s9
AbSTracTion
venture
conflICt
paRticipate
SToRAGE
Pie
LESBIAN
esSEnce
SEVEn40
URANIUM
MANY
VICTIm
supper
2777i3072t
BRIEF
STraIGhT
chEerLEaDEr
dEADLiNE
RarE
COntact
t5L01R12
prObable
cUriOUs
pertaIn
lATITude
MINUS
voyagE
079e
hOLD
t13150935
4O3p7N
Slave
deMonstraTe
proof
ATMOSPHERE
slOPe
SLeeVe
INCLuDING
UNPREDIcTABLE
nEpHew
PrONO6524
cLASsrooM
vaLue
flaTTER
DIMe
suggEst
STaGGER
CLaUsE
blAck
LeaF
rom2Nce
qUiCk
RemeDy
PROPERTy
LINGERING
smALlpOx
STatuS
Comb
accidental
commission
sECUritY
epic
inVasIon
COlLaR
MAGIC
ORDerING
iNveNTION
AddiTIoNaL
emergency
GREATNESS
PART-TIME
TOxIc
house
gARDENER
HAte
POIsOnOuS
cAPABLE
2MP62
ClOThE
8a48a1
lOoSEn
rACE
aWfuL
OFfICe
VENT
OrNaMENT
prOgrEsSIvE
MIstress
handle
nest
TeRN
irritabLe
vertical
JUST
DIGESTION
cry
2R21t03Te
MoDEraTE
cocKpit
TRACE
rusty
opTionaL
DAMe
kItChEN
COsMoS
SATISFY
accurate
KITcHe1
TakE
afraid
reGreT
simultaneously
750d
go
CoMPEtEnCe
monOPoLY
reform
KEEP
STUbBoRn
PUBlIShER
EXPERIMENTaL
SYMpatheTic
HOSe
cHoke
offEnse
sITUAted
HURRICAnE
UNdERGo
c
i
triggEr
sAMe
SOverEiGnTy
l
MINER
CONTROLLER
37641t37
invItatIon
so9
DRAFT
cRACK
FoX
CREDIbLe
advance
couraGe
CoWbOY
I9fan9Ry
212l8T3oN
monstrous
ClouD
CRItICAl
SELL
TourIsT
tie
PursUIT
sUcCeeD
SCARF
hOSpitaBle
CoincidEncE
RIBBoN
hyGIE8e
admiRE
SUITAbLE
unite
SometImeS
microwave
ACcLAiM
LunAr
St7eet
teaC6
SHelteR
ReStORE
evIdENCe
CULTIVATE
im60rA1
resERVE
periodIcal
compromise
PLAnETarY
COnCeNTRaTE
deteriORatE
gram
INVisiBle
c5I3F
AmoUnT
BOaRd
ATteNDANt
tolL
period
JUICE
POnD
pOlE
C9nt3aD2cT
thesIS
Par1n3
blInK
hosTess
NInE
SWAN
ILluMinate
rugged
EEL
72g
adviCe
bUnK
ACCUSATION
ROBE
EXtrAORDINARY
neaT
fur
i1vade
hoMEtOWn
