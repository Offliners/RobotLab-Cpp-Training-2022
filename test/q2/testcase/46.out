n./v. 貨車；載運
n. 碗
n. 美髮師
n. 惡性腫瘤；癌症
v. 提議
v. 使重疊
n./v. 玩笑
adj. 戲劇性的
n. 出席
v. 激怒
v. 堅持
n. 獎券；彩票
adj. 強健的
n./v. 交叉；越過
n. 標題
v. 提升
adj. 裸體的
n. 罐；罈；瓶
Unknown
n. 麵團
prep. 每
adj. 可轉換的
adj. 疲倦的
n./v. 郵件；郵寄
n. 暗喻
n. 禮貌；習俗
Unknown
adj. 無窮的
v.(n. ) 指派
n. 素描；製圖
n. 外科醫師
n./v. 花；開花
adj. 完全地
n. 友誼
adj. 修長的；苗條的
n. 保護
prep./adv./adj.  在…上面；優於，勝過
n. 傳播
n. 連字號
Unknown
n. 雞尾酒
Unknown
n. 階級組織
n. 居民
adv. 有朝一日
Unknown
n. 數量
adj. 自動的
Unknown
adj. 堅韌的
n. 官僚
n. 反抗者
n. 蒙羞
n./v. 味覺；品嚐
v. 召喚
n. 可能性
n. 發明者
n. 長沙發；睡椅
v. 提升
v. 繼續進行
n. 壁櫥；衣櫥
adv. 在·前面；領先
v. 哺乳
adj./v. 正確的；更正
n. 旅；隊
v. 減輕(痛苦)
v. 清理
n. 付費高速公路
n. 苦難
n. 熊貓
v. 假定；假裝
adj. 地理學的
n. 決心
adj./n. 初步的；初步行動
n./v. 命令；控制
n./v. 座位；坐下
n. 應試者
n. 乞丐
n./v. 掙扎；努力
v. 飄浮；搖擺
n. 護士
n. 疾病
v. 見到
Unknown
Unknown
adv. 更好地
Unknown
n. 海灣；深淵
Unknown
n. 失敗者
Unknown
n. 運河
adj. 令人厭煩的
n. 穀粒；穀物
adv. 離開；不在
v. 前往
prep./n./adj. 加
n. 靈魂
n. 庭院
n. 假期
v. 反抗
n. 關聯
v. 編輯
v. 使成為必要
n./adj. 獨唱；單獨的
v. 依從
Unknown
n. 交通
adj. 非正式的
v./n. 再檢查；複習
adj. 活著的
n. 裙子
Unknown
n. 結果
prep. 除了…之外
adj. 華麗的
n. 立法者
Unknown
adv. 笨拙地
adj./adv./n. 二分之一的
n./adj. 釘書針；主要的
v. 死亡
pron. 誰
n./v. 錄音帶
n. 沙拉
n. 鴨子
n. 病毒
v./n. 呻吟(聲)
n.  聽寫；命令
n. 哲學
n. 衣服
v. 競爭；妥善處理
v. 使出糢
adj. 極大的
n. 攝影術
n. 黃瓜
Unknown
n. 鬼；幽靈
adj. 空白處的；邊際的
adj. 潮溼的
n. 方面；觀點
n. 睡衣褲
n. 天使
n. 刺激
adj. 很可能的
n. 浴室
v. 重新開始
adj. 痛苦的
n. 土地
n. 識字；讀寫能力
adj. 顯明的
n. 酒鬼
n. 盤子
n. 痛苦
n. 噴泉；噴水池
n/v. 公告；注意；通知
n. 頭痛
Unknown
n. 赤字
n. 海盜
n. 同輩
n. 胡椒粉
adj. 狂怒的；猛烈的
n. 重新結合；聚會
n. 行星
adj. 使人興奮的
n. 鯉魚
n. 新鮮人
v. 猛衝
Unknown
n. 碟；菜餚
n. 外交官
n. 習慣
Unknown
n. 農人；農場經營者
n./v. 使厭煩
adj./n. 相同的
v./n. 移植
n. 含酒精的飲料
Unknown
adj. 有禮貌的
n. 狂熱的崇拜
n. 儀器；樂器
n. 太太
n. 懇求
n. 春天
adv. 在戶外
v. 把…歸因於
n. 隔離
adj. 突然的
n./v. 請求；要求
n. 櫻桃
n. 保留
n. 航空公司；航線
v. 使逐漸發展
n. 主義
n. 工廠
pron. 誰的
n. 評價
n. 記錄
n./v. 接近；著手處理
n. 救星
adj./n. 喝酒醉的
n. 駱駝
n. 評論家
adj. 生動的
n. 同義詞
adj./n. 黑暗的
n. 書籍
Unknown
n. 生長；發育
v. 用力投擲
n. 攤位
n./v. 行動；表演
n. 教授
adj. 詳細的
n. 難民
v. 使模糊
n. 承認
adj. 充滿希望的
n. 個性
adj. 傳統的
n. 喇嘛
adj. 冗長的
n./v. 注目；注意
n. 警察
adj. 昂貴的
v. 連結
n. 精子
adj. 家常的
n. 公事包
adj. /n易；汽車
adj. 悲慘的
adj. 膽小的
n./v. 太陽；曬太陽
adj. 美麗的
v. 抑制
Unknown
n. 抒情詩
n. 星期六
Unknown
v. 使悲傷
n. 毛巾
adj. 必死的
adj. 值得讚揚的
Unknown
n. 互相對應的人(物)
v. 安慰
n. 小旅館；客棧
n. 後代
adj. 似畫的
v. 達到頂點
v./n. 追捕；打獵
v. 發射
adj./adv./pron. 全部的；全體
adj. 令人驚訝的
v. 使混亂
adv. 永不
n./v. 觸覺；觸摸
adj./adv./n. 南方的
adj. 陰冷(鬱)的
v. 慫恿；引誘
adj. 生氣的
Unknown
adj. 失業的
nJadv. 現在
adj./n. 音樂的
adj. 變老的
adj. 緊密的
adj./n./v. 相反的；反轉
v./n. 交換；交易
n./v. 調動
Unknown
n. 會議
n. 耽溺
n. 緊張
Unknown
Unknown
adj. 因…的結果而起的
v./n. 失敗；不及格
Unknown
n. 哩數
v. 主持
adj./v.  鎮靜的
n. 小冊子
n./v. 經驗
Unknown
n. 關稅
v. 翻譯
n./v. 警報；驚恐
prep./adv. 靠著…旁邊
n. 海灘
adj. 不禮貌的
v. 抓(傷)
n. 邊綠
adj. 基礎的
Unknown
n. 考慮；動機
n./v. 霜；凍
n./v. 圖畫；想像
n. 紀念碑；紀念館
n. 海灣；港灣
Unknown
n. 多樣性
adj./adj. 緊的；緊緊地
adv. 幾乎
n. 奔跑的人
n. 甲狀腺
n. 嗜好
n./v. 齒輪；搭上齒輪
v. 競爭；比賽
n. 顧問
n. 情感；激動
n./v. 氣味；聞
adj.  有效率的
v. 吸引；引誘
n. 舞廳
v. (n)實現
n. 氣色
n. 表演
n. 事情；問題
n. 配偶
Unknown
Unknown
n. 結合
v. 參加
n. 花(叢)
adj. 外部的
n. 養育
n. 廢除
v./n. 展示
n. 偵察機
n./v. 搶救
n. 閃電；閃光
n. 蒙羞
adj. 異性戀的
v.(n. ) 宣布
Unknown
prep./adv. 在遠處
Unknown
n./v. 益處；使用
Unknown
adj. 易變的
Unknown
n. 芥末
v. 後悔
v. 拖
adj. 公平的
adj. 潮溼的
adj. 有教養的
Unknown
adj. 明智的
n. 獎品
n. 海岸巡邏隊
adj. 無效力的
Unknown
v. 烘；烤
n. 大陸；大洲
n. 慈悲
n./v. 鞦韆；搖擺
n. 肢；臂
n. 散文；論說文
Unknown
Unknown
n. 籃子；簍子
v. 跌價
n. 沉思；冥想
pron. 本身
v. 使孤立
Unknown
n. 生物
Unknown
v. 攀爬
n. 反派角色
v. 把…淹死；淹沒
n. 奇蹟
n./v. 尊敬
n. 奢侈
n./v. 打噴嚏
adv. 最後
n./v. 人群；擠滿
n. 墊子
n. 請願(書)
n. 母親；媽媽
adj. 粗糙的
n. 林蔭大道
adj. 重要的
n. 售貨棚
n./v. 法官
n. 雕像
adj. 機密的
n. 午餐
n. 疫苗
n. 連接
n. 醜聞
v. 烤
adj. 多災多難的
n./v. 抵達
adj. 奇怪的
n. 觀察
n. 品脫(液量名)
n./v. 贊同；偏袒
n. 可信性
v. 使驚慌
n. 自助餐；快餐
ad v. 後來
adv. 此外
v. 使多樣化
n. 擦除器(如橡皮擦)
n. 武器
Unknown
n. 無決斷力
n. 豌豆
Unknown
n. 樹脂；口香糖
n. 注意力
adj. 可敬的
adv. 驕傲地
n. 理論
adj. 豐富
n./v. 軌道；追蹤
adj. 暫時的
adj./n. 第三
v. 偷聽；無意中聽到
v. 做不正當的事
n. 航空器；飛機
n. 磅
n./v.  小考；考驗
Unknown
n./v. 晶片；削
v. 爆發；發生
n. 大道
n. 民主主義者
n. 彩虹
Unknown
v. 洗澡；沐浴
n. 澡缸；沐浴
n. 擴散
n. 炊具
adv.  因此
adj./adv./n. 東方的
n./v. 輕跳；略過
n. 週
n. 反語
n. 衝突
adj. 多雨的
n. 外交
n. 日落
adj. 優美的
v.(n. )投資；投入
n. 恐龍
adj./v. 多餘的；撥出
n. 高度；層；水準
n. 威嚴；王權
n. 方言
n. 邏輯
n. 持有者
n. 地主
n. 差異
n. 蓄水池
n. 過渡(期)
n. 低音部
n./v. 跳躍
n. 象徵
Unknown
adj. 毀滅性的
adj. 認真的
n. 音節
adj. 波蘭的；波蘭人(語)的
n. 秋季
n. 建立；基金會
Unknown
adj. 粗俗的
n. 宇宙
adj. 美味可口的
adj. 可怕的
n./v. 損害；毀壞
n. 把手
Unknown
adj.  喜歡的；愛好的
n. 方向
Unknown
v. 分離；啟程
prep./adv. 關於
n. 鐵路
adj. 快的
n. 聯結；契約；債券
n. 獵人
n. 矛盾
n. 羊
n. 二月
n. 記號
n. 電視機
Unknown
adj. 光榮的；輝煌的
n. 准許
adj. 令人厭惡的
n. 對手
n. 綜合(體)；合成
n. 架子
n. 住所
adj. 殘酷的
n. 馬戲團
adj. 不須負責任的
Unknown
n. 結果；作用；影響
n. 救生艇
adj. 可笑的
Unknown
n./adj. 選擇
Unknown
v. 操勞過度
adj. 永久的
adv. 這樣
Unknown
n. 智力商數
n. 自助餐；快餐
n./v. 休息室；懶洋洋地坐或站
v. 敢於…；膽敢
adj./ad v. 來自海外的
v. 不同於…
n. 視力
adj. 運動的
adj./n. 軍事的；軍隊
v. 愛撫
v./n. 避開
adj. 年代久遠的
n. 令人驚奇或驚喜的事物
n. 債
Unknown
adj. 支配的
Unknown
adj. 不貴的
v. 敘述
v. 徘徊(於)
n. 椅子
adj. 堅持的
adj. 嚇壞了的
Unknown
n. 推測
adj. 慣例的；協定上的
adj. 有趣的
n. 角落
pron./conj. 什麼
Unknown
n. 袋子
n. 私生子；令人討厭的人
adj. 幸運的
Unknown
n. 讓步
v. 取回
adj. 香醇的；柔和的；老練的
Unknown
n. 材料
n. 藤條；甘蔗
adj. 合作的
n. 瓷器
adj. 意識形態的
Unknown
adj. 清白的；無知的
n. 敵人
Unknown
n. 照片
n. ＋一月
v./n. 證明有罪；囚犯
Unknown
n. 魔術師
n. 咖啡店；咖啡
Unknown
n. 桃子
n. 百科全書
n. 場所
adj. 防護的；保衛的
n. 恢復
n. 大使；使節
adj. 免除的
n. 驗證；示威運動
Unknown
v. 監督
adv. 仍然；不過
n. 餐巾
n. 紙張
n. 外貌；顯露
adj. 孩子氣的
n. 鏡片；眼球的水晶體
n. 災難
n. (被)起訴
Unknown
adj. 好的
v. 使標準化
n. 附件
n. 效率
adj. 精巧的；複雜的
n. 隧道
n./v. 形狀；做成
n. 街道
v. /n贊助(者)
Unknown
adj. 高貴的；貴族的
n. 成年；男子氣質
n. 橫切
adj. 以前的
adv. 個別地
Unknown
Unknown
n. 心不在焉／抽象概念
n. 冒險(行為)；投機
n./v. 衝突
v. 參與
n. 儲存
n. 派
n. 女同性戀者
n. 本質；精華
Unknown
n. 鈾
adj./pron.  許多
n. 受害者
n. 晚餐
Unknown
adj./n./v. 簡短的
adj./adv. 筆直的
n. 啦啦隊隊長
n. 截止期限
adj. 珍貴的
n./v. 接觸；聯繫
Unknown
adj. 很可能的
adj. 好奇的
v. 有關
n. 緯度
prep./adj./n. 負的
n. 航行
Unknown
v./n. 抓住；支撐
Unknown
Unknown
n.  奴隸
v. 證明；展示；遊行
n. 證據
n. 氣壓；氣氛
n. 斜坡
n. 衣服袖子
prep. 包含·在內
adj. 不可預測的
n. 姪兒；外甥
Unknown
n. 教室
n./v. 價值；估價
v. 諂媚；奉承
n. 一角硬幣
v. 建議
v. 蹣跚
n. 子句；條款
adj./n. 黑的
n. 葉子；一張或一頁
Unknown
adj./adv. 快速的/地
n. 治療
n. 財產
adj. 持久的
n. 天花
n. 社會地位
n./v. 梳子；梳頭
adj. 偶然的；意外的
n./v. 佣金；委託
n. 安全
n./adj. 史詩(般的)
n. 入侵
n. 衣領；項圈
adj./n. 有魔力的；魔法
n. 安排；佈置
n. 發明；創造力
adj. 附加的
n. 緊急情況
n. 很多
adj./adv. 兼任的
adj. 有毒的
n./v. 住宅
n. 園丁；花匠
v./n. 憎恨
adj. 有毒的
adj. 有能力的
Unknown
v. 為…穿衣
Unknown
v. 放鬆
n. 比賽
adj. 糟透的
n. 辦公室
n. 通風孔
n. 裝飾
adj. 不斷前進的
n. 女主人
n. 柄；把手
n./v. 築巢
n. 期限
adj. 急躁的
adj. 垂直的
adj. 公平的
n. 消化作用
v./n. 哭；叫喊
Unknown
adj. 中等的
n. 駕駛艙；鬥雞場
n./v. 蹤跡；追溯
adj. 的鐵釘
adj. 可選擇的；隨意的
n. 貴夫人
n. 廚房
n. 宇宙
v. 滿意
adj. 準確的
Unknown
v. 拿(走)
adj. 害怕的
n./v. 哀傷；遺憾
adv. 同時
Unknown
v. 去；從事
n. 能力
n. 壟斷；專利
v. 改革
v. 保存
adj. 固執的
n. 出版社(者)
adj. 實驗性質的
adj. 有同情心的
n. 長統襪及短襪
v./n. 窒息；阻塞
n. 罪過；冒犯
adj. 坐落在…的
n. 颶風；暴風雨
v. 接受；經歷
Unknown
Unknown
n./v. 扳機；引發
adj./pron. 同樣的
n. 主權
Unknown
n. 礦工
n. 控制者
Unknown
n. 邀請
Unknown
n./v. 草稿
n./v. 爆烈聲；裂開
n. 狐狸
adj. 可信的
n./v. 推進；將…提前
n. 勇氣
n. 牛仔；牧牛人
Unknown
Unknown
adj. 怪物的；恐怖的
n./v. 雲；遮蔽
adj. 關鍵性的；批判的
v. 賣
n. 觀光客
n. 領帶
n. 追趕
v. 達成
n. 領巾
adj. 好客的
n. 巧合
n. 絲帶
Unknown
v. 欽佩；欣賞
adj. 合適的
v. 合併
adv. 有時
n. 微波
v./n. 歡呼；歡呼聲
adj. 陰曆的；月亮的
Unknown
Unknown
n./v. 避難所
v. 歸還
n./v. 證據；物證
v. 耕作；培養
Unknown
v. 預備
n. 期刊
v. 妥協(方案)
adj. 行星的
v. 專心
v. 惡化
n. (重量單位)克
adj. 看不見的
Unknown
n./v. 共計；總數
n. 木板；牌子
n. 服務員
n. 通行費
n. 一段時問
n. 果汁
n. 池塘
n. 南極或北極
Unknown
n. 論文
Unknown
v. 眨眼；漠視
n. 女主人
adj./n. 九個
n. 天鵝
v. 照亮
adj. 崎嶇的
n. 鰻
Unknown
n. 勸告；消息
n. 臥鋪
n. 控訴
n. 長袍
adj. 異常的
adj. 整潔的
n. 毛皮
Unknown
n. 永久居留地
